index.js

```diff
@@ -3,30 +3,43 @@
 const semver = require('semver')
 const console = require('console')
 
-function plugin (fn, version) {
+function plugin (fn, options) {
```

Comment:
I would keep it backward-compatible.

Reply:
It is https://github.com/fastify/fastify-plugin/blob/88997b05faf79c3a4d9b777d4d474b0f3eeb83a1/index.js#L15

Reply:
this is not true anymore, correct?

Reply:
Correct.